style.css

```diff
@@ -660,6 +758,144 @@
     min-height: 30%;
   }
 
+  .modal {
+    border: 2px rgba(255, 255, 255, 0.521);
+    width: 80%;
+    border-radius: 15px;
+    height: 80%;
+  }
+
+  .modal::backdrop {
+    background: #a7a7a79c;
+    backdrop-filter: blur(2px);
+  }
+
+  .styled-modal {
+    justify-content: space-between;
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+  }
+
+  h2 {
+    margin: 0;
+  }
+
+  .tech-links-modal div {
+    display: flex;
+    flex-direction: row;
+    align-content: center;
+  }
+
+  .styled-modal h2 {
+    padding-left: 20px;
+    font-family: Poppins-Bold, sans-serif;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  #img-modal {
+    width: 80%;
+    height: 100%;
+    padding-left: 10%;
+  }
```

Comment:
- [ ] Good job so far 👍👍 However, as suggested by the reviewer [here](https://github.com/SaveryIV/SaveryIV.github.io/pull/20#discussion_r1235073151), kindly make sure that the image spans the width of the container as given in the design guidelines ✌✨ for getting a design close to the Figma template as much as possible


Your design | Given design 
---| ---|
![image](https://github.com/SaveryIV/SaveryIV.github.io/assets/106119105/b6e45b2d-2440-4056-9a70-32695e9346b9)  | ![image](https://github.com/SaveryIV/SaveryIV.github.io/assets/106119105/8be247a5-cd55-48ba-9c66-cbfe3a38fc66)